examples/angular/README.md

```diff
@@ -1,64 +1,153 @@
-3. Node js 18+ and npm installed
-4. Terminal
-5. And a code editor.
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+  - [Obtaining the Example Code](#obtaining-the-example-code)
+  - [Configuration](#configuration)
+- [Running the Application](#running-the-application)
+- [Project Structure](#project-structure)
+- [Key Features](#key-features)
+  - [Handling Vanity URLs](#handling-vanity-urls)
+- [Troubleshooting](#troubleshooting)
+- [Further Resources](#further-resources)
 
-### Get the Angular example code
+## Prerequisites
 
-Get the code from the Angular example directory
+Before you begin, ensure you have the following:
 
-```bash
-https://github.com/dotCMS/core/tree/master/examples/angular
-```
+1. Access to a dotCMS instance (you can use https://demo.dotcms.com if you don't have your own)
+2. A valid AUTH token for the target dotCMS instance ([How to create an API token](https://auth.dotcms.com/docs/latest/rest-api-authentication#creating-an-api-token-in-the-ui))
+3. Node.js (version 18 or higher) and npm installed
+4. A terminal application
+5. A code editor of your choice
 
-Or just checkout the directory
+## Getting Started
 
-```bash 
-git clone -n --depth=1 --filter=tree:0 https://github.com/dotCMS/core
-cd core
-git sparse-checkout set --no-cone examples/angular
-git checkout
-```
-The files will be found under the `examples/angular` folder
+### Downloading the example
+
+You can get the code in two ways:
+
+1. Direct download:
+   ```
+   https://github.com/dotCMS/core/tree/master/examples/angular
+   ```
+
+2. Using Git sparse checkout:
+   ```bash
+   git clone -n --depth=1 --filter=tree:0 https://github.com/dotCMS/core
+   cd core
+   git sparse-checkout set --no-cone examples/angular
+   git checkout
+   ```
+   The example files will be in the `examples/angular` folder.
 
+### Configuration
 
+To configure the Angular app to use your dotCMS instance:
 
-## Add the dotCMS configuration
+1. Open the project folder in your code editor
+2. Navigate to `src/environments`
+3. Open `environment.development.ts` and update the following variables:
+   - `authToken`: Your dotCMS auth token
+   - `dotcmsUrl`: URL of your dotCMS instance (e.g., https://demo.dotcms.com)
 
-Now we need to tell the Angular app what dotCMS instance is going to use to get the data to build its pages.
+   ```typescript
+   export const environment = {
+     production: false,
+     authToken: 'YOUR_AUTH_TOKEN_HERE',
+     dotcmsUrl: 'https://demo.dotcms.com',
+     // other configurations...
+   };
+   ```
 
-1. Open the folder `YOUR_NAME` in your code editor
-2. Go to `src/environments`
-3. Open the `environment.development.ts` file and update the environment variable:
+   ⚠️ **Security Note**: Ensure that the `authToken` used here has read-only permissions to minimize security risks in client-side applications.
```

Comment:
We need a link here to the permissions docs? Did we have something specific for this? I remeber you wrote something, not sure if Jamie added somewhere.